sdcm/nemesis.py

```diff
@@ -1159,7 +1160,8 @@ def add_ldap_configuration_to_node(node):
             raise LdapNotRunning("LDAP server was supposed to be running, but it is not")
 
     @retrying(n=3, sleep_time=60, allowed_exceptions=(NodeSetupFailed, NodeSetupTimeout))
```

Comment:
@fruch , @fgelcer , following new usage of hard-timeout - we can consider removing this retry.

Reply:
we shouldn't consider this, for `replace_node` case we shouldn't be doing retries

Reply:
@yarongilor ?